advanced.md

```diff
@@ -167,4 +167,112 @@ public void toggleAllDigitalPins( bool setPinsHigh )
 		firmata.sendDigitalPort( 1, 0 );	//all pins low
```

Comment:
Without PinConfig implemented, we won't know how many port defines are valid, and the '0' represents the state of 8 pins as one pin per bit.